cfme/tests/containers/test_ssl_providers.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+from itertools import product
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.provider.openshift import OpenshiftProvider
+from utils import testgen
+from utils.version import current_version
+from utils.appliance.implementations.ui import navigate_to
+from utils import conf
+from cfme.web_ui import fill, flash
+from utils.providers import get_crud
+from utils.ssh import SSHClient
+from cfme.fixtures import pytest_selenium as sel
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.8.0.3")]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='module')
+
+provider_name = conf.cfme_data['management_systems']['cm-env1'].name
+DEFAULT_SEC_PROTOCOLS = ['SSL trusting custom CA', 'SSL without validation', 'SSL']
+HAWKULAR_SEC_PROTOCOLS = ['SSL trusting custom CA', 'SSL without validation', 'SSL']
+
+
+@pytest.mark.parametrize('default_sec_protocols', product(DEFAULT_SEC_PROTOCOLS))
```

Comment:
REQUIRED: what for do you create a list of tuples and then make them list of strings again ?
I suppose you can just remove both product and "".join and pass just ordinary list/tuple.